LoopKit/QuantityFormatter.swift

```diff
@@ -195,19 +195,23 @@ public extension HKUnit {
     }
 
     var pickerFractionDigits: Int {
+        maxFractionDigits
+    }
+    
+    var maxFractionDigits: Int {
         switch self {
         case .internationalUnit(), .internationalUnitsPerHour:
             return 3
         case HKUnit.gram().unitDivided(by: .internationalUnit()):
             return 1
-        case .millimolesPerLiter,
-             HKUnit.millimolesPerLiter.unitDivided(by: .internationalUnit()),
-             HKUnit.millimolesPerLiter.unitDivided(by: .minute()):
-            return 1
         default:
-            return 0
+            return preferredFractionDigits
         }
     }
+    
+    private var resolution: Double {
```

Comment:
"Resolution" doesn't make sense to me as property of an HKUnit. It seems wrong to say (as this method name suggests) that International Units has a resolution of 0.001 U.  I'd recommend either inlining it into the method it's being used in, or qualifying it like `resolutionUsingMaxFractionDigits`.  I think the former would be preferred, especially if we want to generalize the `allValues` method to not assume `maxFractionDigits`, which seems useful.

Reply:
hm... generalizing allValues and passing in the increment was one of the concerns you raised about how allValues was being derived. Or do I have a misunderstanding.

I'll use `resolutionUsingMaxFractionDigits`.

Reply:
My original concern with `allValues...` was a mismatch between the passed in stride and the internal use of a fixed number of digits.  As long as the striding and rounding are based on the same precision, then parameterization of the method is fine, and actually preferred, imo.

Reply:
ah... got it. The rounding will also use the passed in fraction digits.